diff --git a/Makefile.am b/Makefile.am
index 2252148..f43d5d5 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -29,7 +29,7 @@ if TESTS
 tests_dir = tests
 endif
 
-SUBDIRS = po doc intl src $(tests_dir)
+SUBDIRS = doc src $(tests_dir)
 
 EXTRA_DIST = AUTHORS.asciidoc \
              ChangeLog.asciidoc \
diff --git a/configure.ac b/configure.ac
index 8e0f242..3ae003d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -59,8 +59,6 @@ esac
 
 # Gettext
 ALL_LINGUAS="cs de es fr hu it ja pl pt_BR ru tr"
-AM_GNU_GETTEXT
-AM_GNU_GETTEXT_VERSION([0.18])
 
 # Checks for libraries
 AC_CHECK_LIB(ncurses, initscr, LIBNCURSES_FOUND=1, LIBNCURSES_FOUND=0)
@@ -69,6 +67,7 @@ AC_CHECK_LIB(ncursesw, initscr, LIBNCURSESW_FOUND=1, LIBNCURSESW_FOUND=0)
 # Checks for header files
 AC_HEADER_STDC
 AC_CHECK_HEADERS([libintl.h sys/resource.h])
+AC_CHECK_HEADER([langinfo.h], [AC_DEFINE([HAVE_LANGINFO_CODESET], [1], [Define if langinfo.h exists.])])
 
 # Checks for typedefs, structures, and compiler characteristics
 AC_HEADER_TIME
@@ -1314,9 +1313,7 @@ AC_OUTPUT([Makefile
            src/plugins/xfer/Makefile
            src/gui/Makefile
            src/gui/curses/Makefile
-           tests/Makefile
-           intl/Makefile
-           po/Makefile.in])
+           tests/Makefile])
 
 # ------------------------------------------------------------------------------
 #                                  end message

