diff --git a/Makefile.am b/Makefile.am
index 2252148..cf2b967 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -29,7 +29,7 @@ if TESTS
 tests_dir = tests
 endif
 
-SUBDIRS = po doc intl src $(tests_dir)
+SUBDIRS = doc src $(tests_dir)
 
 EXTRA_DIST = AUTHORS.asciidoc \
              ChangeLog.asciidoc \
diff --git a/configure.ac b/configure.ac
index 86272a7..9e4f123 100644
--- a/configure.ac
+++ b/configure.ac
@@ -59,8 +59,6 @@ esac
 
 # Gettext
 ALL_LINGUAS="fr es cs hu de ru pl it ja pt_BR tr"
-AM_GNU_GETTEXT
-AM_GNU_GETTEXT_VERSION([0.18])
 
 # Checks for libraries
 AC_CHECK_LIB(ncurses, initscr, LIBNCURSES_FOUND=1, LIBNCURSES_FOUND=0)
@@ -1314,9 +1312,7 @@ AC_OUTPUT([Makefile
            src/plugins/xfer/Makefile
            src/gui/Makefile
            src/gui/curses/Makefile
-           tests/Makefile
-           intl/Makefile
-           po/Makefile.in])
+           tests/Makefile])
 
 # ------------------------------------------------------------------------------
 #                                  end message
